package.json

```diff
@@ -89,7 +89,7 @@
     "gulp-rollup-lightweight": "^1.0.10",
     "gulp-rtlcss": "^1.4.2",
     "gulp-sass": "^5.1.0",
-    "gulp-uglify-es": "^2.0.0",
+    "gulp-uglify-es": "^3.0.0",
```

Comment:
since we use this dep for bundling, just wanted to double check that you tried making running a build and using the bundle locally to test that it still works? wasn't sure how big of a difference there was in this library since we're bumping to a new major version

Reply:
actually, I'm not seeing any of the vulnerability items pointing specifically to gulp-uglify-es. if this is for the terser vulnerability, can we just `npm update terser`?

Reply:
updated